multiview/embed/mvmds.py

```diff
@@ -0,0 +1,216 @@
+# Copyright 2019 NeuroData (http://neurodata.io)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from multiview.embed.base import BaseEmbed
+from multiview.utils.utils import check_Xs
+import warnings
+import numpy as np
+from sklearn.metrics import euclidean_distances
+
+
+class MVMDS(BaseEmbed):
+
+    """"
+    An implementation of Classical Multiview Multidimensional Scaling for
+    jointly reducing the dimensions of multiple views of data. A Euclidean
+    distance matrix is created for each view, double centered, and the k l
+    argest common eigenvectors between the matrices are returned based on
+    the stepwise estimation of common principal components.
+
+    Parameters
+    ----------
+    n_components : int (positive), default=None
+        Represents the number of components that the user would like to
+        be returned from the algorithm. This value must be greater than
+        0 and less than the number of samples within each view.
+
+    num_iter: int (positive), default=15
+        Number of iterations stepwise estimation goes through. Detailed
+        in Trendafilov paper.
+
+    Attributes
+    ----------
+    components: A projection of shape [n_samples, n_components]
```

Comment:
Format this correctly (components: return type and then description below). Having the shape here is good